### DISCUSSION TITLE
lldb-mi?

### DISCUSSION THREAD

**Discussion by JetForMe** [WebFreak001/code-debug#392] [Q&A] [answered]
`lldb-mi` doesn't seem to be included in Xcode 14, and I can't even find it in the official LLVM sources. There is [this](https://github.com/lldb-tools/lldb-mi) repo, but that doesn't seem to be official.

**Comment by GitMensch** [comment, marked as answer]
That _is_ the official repo, see its announcement in https://discourse.llvm.org/t/lldb-mi-has-been-moved-to-its-own-github-repository/52817

**Comment by JetForMe** [comment]
Oh, thank you for that.
